tests/tests.cpp

```diff
@@ -151,6 +151,13 @@ int main() {
   user_defined_container<int, 3> dummy_udc = {{11, 22, 33}};
   dbg(dummy_udc);
 
+  dbg(std::pair<int, bool>{13, true});
+
+  std::pair<std::pair<bool, int>, bool> pair_of_pairs{{false, 17}, true};
+  dbg(pair_of_pairs);
+
+  dbg(std::pair<std::string, int>{"Hello", 7});
+
```

Comment:
It would be great if we could move these to the actual tests-section of this file (I should really clean this up a little bit) and formulate them as tests. See the "pretty_print tests" section below.